sdk/textanalytics/azure-ai-textanalytics/azure/ai/textanalytics/_text_analytics_client.py

```diff
@@ -93,6 +93,7 @@ def __init__(self, endpoint, credential, **kwargs):
         )
         self._default_language = kwargs.pop("default_language", "en")
         self._default_country_hint = kwargs.pop("default_country_hint", "US")
+        self._string_index_type = None if kwargs.get("api_version") == "v3.0" else "UnicodeCodePoint"
```

Comment:
would there be any possibility that in the future the user will be able to change this?
My thinking is.. if we should have this as `default_string_index_type`

Reply:
this property isn't exposed to users, so I can change this code anytime (it's just an internal property to keep track of the python encoding). If in the future we do decide to expose the param with name `string_index_type` to users, this won't interfere at all

Reply:
I understand. I am wondering if we should append `default` to the name, even internally, for consistency with the other properties and to tell whoever reads/works on this code later, that this is the default value assigned for the client